src/components/SignalCard/card/SignalCard.module.scss

```diff
@@ -263,3 +295,13 @@
   right: 16px;
   bottom: 24px;
 }
+
+.frozenAlert {
+  height: 32px;
+  padding: 8px 12px;
+  background: var(--frozen-alert-bg, var(--white));
+  border: 1px solid var(--porcelain);
+  border-radius: 4px;
+  gap: 8px;
+  fill: var(--waterloo);
```

Comment:
Use [`san-webkit`'s](frozenAlert) `.btn-2.btn--s` classes. Add `mrg-s mrg--r` to the icon element